yarn.lock

```diff
@@ -13706,7 +13760,25 @@ tslib@^1.8.0, tslib@^1.8.1, tslib@^1.9.0, tslib@^1.9.3:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.9.3.tgz#d7e4dd79245d85428c4d7e4822a79917954ca286"
   integrity sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ==
 
-tslint@^5.11.0, tslint@^5.9.1:
+tslint@^5.11.0:
```

Comment:
Not sure how we pulled in two `tslint` versions. Can you remove one entry and run `yarn` again? Or fiddle around with the version required in the workspace root?

Reply:
fixed